### ISSUE TITLE
Every email is send to the spam folder why and how to fix it?

### ISSUE THREAD

**Issue by CharlesGruff** [WPManageNinja/fluent-smtp#143]
hi im creating a wordpress site and i installed SMTP to receive emails when i use the contact form for exemple. Unfortunaly, every emails is redirected to the SPAM folder

**Comment by Alvarnegro** [comment]
I have the same problem... Since installing the plugin, all emails end up in spam...
Now, if I deactivate or delete the plugin, the emails still arrive in the spam folder.

We have spoken with our hosting and the email system is fine, the website files do not show malware and in principle everything is correct. Except because the emails sent by the website do not arrive.

Here is a summary of the body of one of the returned messages:
======================

This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its recipients. This is a permanent error.

The following address failed:

    [escuela@parapentevejer.com](mailto:escuela@parapentevejer.com):
        virus/suspect content found

--- The header of the original message is following. ---

Received: from parapentevejer.com ([82.165.84.48]) by mrelayeu.kundenserver.de
 (mreue106 [213.165.67.118]) with ESMTPSA (Nemesis) id
 1M5xDJ-1qwq1F3O3P-007RI0 for <[escuela@parapentevejer.com](mailto:escuela@parapentevejer.com)>; Wed, 11 Oct 2023
 11:31:34 +0200
Date: Wed, 11 Oct 2023 09:31:34 +0000
To: [escuela@parapentevejer.com](mailto:escuela@parapentevejer.com)
From: Parapente Vejer <[vuelacon@parapentevejer.com](mailto:vuelacon@parapentevejer.com)>
Subject: Fluent SMTP: Test Email - Parapente Vejer - HTML Version
Message-ID: <[dIhAbcXW5ksqLUTXZQBIMQKrVC2Ba2oMebJw8FQQro@parapentevejer.com](mailto:dIhAbcXW5ksqLUTXZQBIMQKrVC2Ba2oMebJw8FQQro@parapentevejer.com)>
X-Mailer: PHPMailer 6.8.0 (https://github.com/PHPMailer/PHPMailer)
MIME-Version: 1.0
Content-Type: text/html; charset=UTF-8
X-Provags-ID: V03:K1:eZ/n/qnzzTf/XFkVBm5fh/A4ZXRv7Ta2KIMc7hMb+2UplBSi7vc
 cZH4btJ5/XvkLYt3fYYDbShrsuMZePZmDK1ILlVU+7c3FBbU2LpZ8bsVlYB2lvK5QA4mYIy
 4xungNFJogiiv/2lzFS7+gYqiAVjHvTkjFcAP/eR2GUt9N7kRhEy0rPw4YSZlL4Mrre9Mqm
 nv+SiAC50oWOWTerKfRwA==

**Comment by techjewel** [comment]
@Alvarnegro This is from your email hosting service. Basically this emails was rejected from the receiver server and for that you got notified. This email is not generated from your WordPress

**Comment by Alvarnegro** [comment]
Yes, the hosting service... says that the email is not built correctly.

The bounce message that you send us has been produced because suspicious content has been detected in the destination mailbox (whichever it is) in the email sent:

This did not happen to us before installing Fluent SMTP

**Comment by Alvarnegro** [comment]
@CharlesGruff Did you manage to solve it in your installation?

**Comment by shoulders** [comment]
dont use PHPMailer, send(), mail() as all emails services give this MTA a lot of spam points.

solution = use a proper mail server or SMTP (setup correctly with SPF and DKIM)
